Testing/ContinuousIntegration/AzurePipelinesLinuxLegacyRemoved.yml

```diff
@@ -0,0 +1,83 @@
+name: ITK.Linux
```

Comment:
The name should be different so that the build can be distinguished from the regular `ITK.Linux`, maybe `ITK.Linux.LegacyRemove` or `ITK.Linux.LegacyRemoved` would honor its purpose.

Reply:
Complete oversight on my part.